packages/worker/src/debug-replayer/client.ts

```diff
@@ -44,7 +44,14 @@ export class Client {
       request.once('response', resolve);
     });
     if (response.statusCode !== 200) {
-      throw new Error(`Bad response code from VS Code: ${response.statusCode}`);
+      const responseBody = await Client.readAll(response);
```

Comment:
I hate to do request another change but can you please move reading the body to the try block?
Not super critical, I'm approving the PR and leaving it up to you.

Reply:
Done, thank you ✅